mautrix_signal/signal.py

```diff
@@ -175,6 +175,12 @@ async def handle_message(
                 )
                 return
         assert portal
+
+        # Handle the user being removed from the group.
+        if msg.group_v2 and msg.group_v2.removed:
+            await portal.handle_signal_kicked(user, sender)
```

Comment:
The room isn't guaranteed to exist here

```suggestion
            if portal.mxid:
                await portal.handle_signal_kicked(user, sender)
```